src/comments/Comments.js

```diff
@@ -26,7 +26,7 @@ export default class Comments extends Component {
   }
 
   static propTypes = {
-    postId: PropTypes.string.isRequired,
+    postId: PropTypes.number.isRequired,
```

Comment:
postId is not a number but a string because we use it as an important object key in our state. API is returning number in some cases and that's why there is a react warning, however in some other cases postId is like this: `111.11.22`.

If you want to remove that warning you should make sure postId is always converted to string before passing as props. We should change the data to fit our requirement not the other way.